static/app/components/nav/submenu.tsx

```diff
@@ -1,39 +1,16 @@
-import {Fragment} from 'react';
+import {useCallback, useMemo} from 'react';
 import styled from '@emotion/styled';
+import type {LocationDescriptor} from 'history';
 
-import Feature from 'sentry/components/acl/feature';
 import InteractionStateLayer from 'sentry/components/interactionStateLayer';
 import Link from 'sentry/components/links/link';
 import {useNavContext} from 'sentry/components/nav/context';
-import type {NavSubmenuItem} from 'sentry/components/nav/utils';
-import {
-  isNavItemActive,
-  isNonEmptyArray,
-  makeLinkPropsFromTo,
-} from 'sentry/components/nav/utils';
 import {space} from 'sentry/styles/space';
-import {useLocation} from 'sentry/utils/useLocation';
-
-function Submenu() {
-  const nav = useNavContext();
-  if (!nav.submenu) {
-    return null;
-  }
 
+function Submenu({children}) {
```

Comment:
This component is now driven by declarative components rather than derived from the config object, which allows custom behavior to be added for specific product verticals.